caffe2/operators/conv_op.h

```diff
@@ -19,8 +19,9 @@ class ConvOp final : public ConvPoolOpBase<Context> {
     // Since this is the default convolution implementation, we will
     // use CAFFE_ENFORCE instead of OPERATOR_NEEDS_FEATURE.
     CAFFE_ENFORCE(
-        group_ == 1 || order_ == StorageOrder::NCHW,
-        "Group convolution only supports NCHW order right now.");
+        (group_ == 1 || order_ == StorageOrder::NCHW ||
+         std::is_same<Context, CPUContext>::value),
+        "Group convolution only supports NCHW order or CPUContext right now.");
```

Comment:
Hi @jspark1105 , I guess what you are trying to comment is "Group convolution only supports NHWC order or CPUContext right now", right?